NitroxModel/Helper/NitroxUser.cs

```diff
@@ -100,5 +100,31 @@ public static string GamePath
                 GamePlatform = GamePlatforms.GetPlatformByGameDir(gamePath);
             }
         }
+
+        public static bool? IsNewestSubnautica
+        {
+            get
+            {
+                if (string.IsNullOrWhiteSpace(GamePath))
+                {
+                    return null;
+                }
+
+                string streamingAssetsFolder = Path.Combine(GamePath, "Subnautica_Data", "StreamingAssets");
+                string aaFolder = Path.Combine(streamingAssetsFolder, "aa");
+
+                if (!Directory.Exists(streamingAssetsFolder)) {
+                    // Probably authorization exception
+                    return null;
+                }
+
+                if (File.Exists(Path.Combine(aaFolder, "catalog.json")) && File.Exists(Path.Combine(aaFolder, "settings.json")))
```

Comment:
If we are just checking that the settings and catalog exist as files is there any point checking the folder exists and not just checking the final path? I would assume if the folder doesn't exist it will throw an error saying no file in the file check as it won't be able to find the paths

Reply:
No error is thrown by File.Exist in any way